Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.9.0-buster
+
+RUN sed -i 's/deb.debian.org/mirrors.aliyun.com/g' /etc/apt/sources.list
+RUN sed -i 's/security.debian.org/mirrors.aliyun.com/g' /etc/apt/sources.list
+
+RUN pip install --upgrade pip && pip install mkdocs mkdocs-material
+
+WORKDIR /app
```

Comment:
Could we use `hello-algo` instead of `app`?

Reply:
YES, we can. But it is unnecessary for it is stored inside the **docker**. Nobody can see its name unless use `docker exec -it xxx` to get a shell in the docker. `app` is the tradition name for python service.